projects/packages/my-jetpack/_inc/components/my-jetpack-tab-panel/products/utils.ts

```diff
@@ -112,3 +115,28 @@ export function filterAndSortModules(
 
 	return $modules;
 }
+
+/**
+ * Get the status of a product based on its availability.
+ *
+ * @param {ProductCamelCase} product - The product to check.
+ *
+ * @return True if the product is supported, false otherwise.
+ */
+export function getProductStatus( product: ProductCamelCase ) {
```

Comment:
This function looks largely similar to getModuleStatus().

Could we abstract some part of the logic for both of them?

Reply:
That abstraction will result in a convoluted logic because product cards and modules are different and will make it difficult to add more conditions for that status if needed in the future.